internal/retry/sizer.go

```diff
@@ -0,0 +1,60 @@
+package retry
+
+// Hint is a type for retry hints
+type Hint interface {
+	size() int
+}
+
+type hintInt int
```

Comment:
Wouldn't `intHint` sound better than `hintInt`?
(Just Kidding! ;-))

Reply:
It was `intHint` before, but then I saw it's `errorString`, not `stringError`... so I changed it. :rofl: